Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orient_polygon_soup.h

```diff
@@ -63,6 +71,7 @@ struct Polygon_soup_orienter
   Edge_map edges;             //< the set of edges of the input polygons
   Marked_edges marked_edges;  //< the set of singular edges or edges incident
                               //<   to non-compatible orientation polygons
+  Visitor& visitor;
```

Comment:
@sloriot do we want a reference or a copy?

Reply:
This is now a pointer.

Reply:
How is that documented?

Reply:
It isn't yet. I'd like to have the code reviewed first, then if we agree on the implem, I'll doc it and make a SF

Reply:
But how is it documented for other visitors in PMP? Holding a ref or a pointer opens the risk of a dandling pointer/ref. If the implementation stays as it is, then in the main function the parameter for the visitor should be a non-const ref.

Reply:
In PMP visitors are passed through NPs and copied.

Reply:
If we do it as in the BGL, visitors are copied and hence must be lightweight.